src/components/dialogs/incorporate-now-error.vue

```diff
@@ -33,6 +34,11 @@ export default class IncorporateNowErrorDialog extends Vue {
 
   /** Pass click event to parent. */
   @Emit() close () { }
+
+  /** Try again button clicked. Refresh the page. */
+  tryAgain () {
+    window.location.reload()
```

Comment:
This seems harsh. Does it work or did the session storage data get nuked already?

What do other error modals do?

Reply:
I've tested and it works! At this point in time, the `LEGAL_TYPE` session storage item is still there. So, upon a refresh, `App.vue` sees the item and moves forward with the regular process.